cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -106,6 +107,10 @@ func (a *StaticAutoscaler) RunOnce(currentTime time.Time) errors.AutoscalerError
 		glog.Errorf("Failed to list ready nodes: %v", err)
 		return errors.ToAutoscalerError(errors.ApiCallError, err)
 	}
+	// Handle GPU case - allocatable GPU may be equal to 0 up to 15 minutes after
+	// node registers as ready.
```

Comment:
Can you link the bug on kubernetes/kubernetes?

Reply:
Done.